packages/cube/lib/namespaces.js

```diff
@@ -0,0 +1,6 @@
+import $rdf from '@zazuko/env-node'
+
+const cube = $rdf.namespace('https://cube.link/')
+const { rdf, rdfs, sh, xsd, _void, dcat, schema, dcterms } = $rdf.ns
+
+export { cube, rdf, rdfs, sh, xsd, _void, dcat, schema, dcterms }
```

Comment:
You can remove this once you use `this.env.ns` which contains all namespaces, including `cube`